configure.ac

```diff
@@ -144,6 +144,11 @@ AC_ARG_ENABLE([runautogen],
 	[ enable_runautogen="no" ])
 AM_CONDITIONAL([BUILD_RUNAUTOGEN], [test x$enable_runautogen = xyes])
 
+AC_ARG_ENABLE([debuginfo],
+	[AS_HELP_STRING([--enable-debuginfo],[build debuginfo packages])],,
```

Comment:
can you please change the name from --enable-debuginfo to --enable-rpm-debuginfo ?